src/Utils/errorHandler.ts

```diff
@@ -0,0 +1,10 @@
+class ErrorHandler extends Error {
```

Comment:
Hey @miami78, great work. I know TS hasn't been smooth on you. Could we have this renamed to `ErrorResponse` instead? I feel like `ErrorHandler` is how you name an error-handling route.

Reply:
@yegow  you are right,let me change that